examples/v1beta1/kind-cluster/deploy.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to deploy Kind cluster with Katib standalone components.
+
+# Verify that appropriate tools are installed.
+if [[ ! $(which docker) ]]; then
+  echo "Unable to find Docker"
+  echo "To install Docker, please follow this guide: https://docs.docker.com/get-docker"
+  exit 1
+fi
+
+if [[ ! $(which kind) ]]; then
+  echo "Unable to find Kind"
+  echo "To install Kind, please follow this guide: https://kind.sigs.k8s.io/docs/user/quick-start/#installation"
+  exit 1
+fi
+
+if [[ ! $(which kubectl) ]]; then
+  echo "Unable to find kubectl"
+  echo "To install kubectl, please follow this guide: https://kubernetes.io/docs/tasks/tools/#kubectl"
+  exit 1
+fi
+
+# Step 1. Create Kind cluster.
+kind create cluster
```

Comment:
I think we should specify node version using `--image` (ex: `--image kindest/node:v1.18.19`) since we do not test all Kubernetes versions in e2e.
WDYT @andreyvelich?

Reply:
Yes, it makes sense. Thanks.